app/src/test/java/org/simple/clinic/scanid/ScanSimpleIdUpdateTest.kt

```diff
@@ -276,4 +276,20 @@ class ScanSimpleIdUpdateTest {
             )
         )
   }
+
+  @Test
+  fun `When complete medical records are saved and one patient is found, open patient summary`() {
```

Comment:
Not that important I think.. We follow `when this, then do that` format in update tests. Can we rename this test to follow the same?
`When complete medical records are saved and one patient is found, then open patient summary`

Reply:
Also, we are changing two update branches here for `OnlinePatientLookupWithIdentifierCompleted` and `CompleteMedicalRecordsSaved` can we separate these out into two commits and write update tests for both the changes? Currently we've only written for when `CompleteMedicalRecordsSaved`

Reply:
* I think comma separation does the job for then but I'll still add this here. 
* No we're not doing two branches here. Completed and saved is written for the better context readability. There are tests already present for `OnlinePatientLookupWithIdentifierCompleted` in the previous PR. Can you PTAL?

Reply:
We added `is Found -> next(model.notSearching(), SaveCompleteMedicalRecords(event.result.medicalRecords))` instead of `else -> noChange() // update the list of patients or patient received here` in `onlinePatientLookupWithIdentifierCompleted` which is a part of `is OnlinePatientLookupWithIdentifierCompleted` branch. Which is why I said we might need one more test as we are doing something new here.